test/e2e/util.go

```diff
@@ -571,6 +536,68 @@ func RunRC(c *client.Client, name string, ns, image string, replicas int) error
 	return nil
 }
 
+func ResizeRC(c *client.Client, ns, name string, size int) error {
+	By(fmt.Sprintf("Resizing replication controller %s to %d", name, size))
```

Comment:
If you could reuse the kubectl libraries: 
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubectl/resize.go
that would be ideal

eg: 
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubectl/stop.go#L82